src/matcha_ml/cli/ui/spinner.py

```diff
@@ -30,9 +30,14 @@ def __init__(self, status: str):
         )
         self.progress.add_task(description=status, total=None)
 
-    def __enter__(self) -> None:
-        """Call when a spinner object is created using a `with` statement."""
+    def __enter__(self):  # type: ignore
```

Comment:
Nit: Just a thought - you could type this return with TypeVar or with a string.

Reply:
Good idea, I'll give it a try 👍🏻